### ISSUE TITLE
oxygen-woocommerce woo-global-styler.css custom checkbox not working

### ISSUE THREAD

**Issue by webzstore-sahel** [soflyy/oxygen-bugs-and-features#2918]
In oxygen-woocommerce 2.0 you have introduced a new feature but it is not working in our environment.
/* Custom Checkbox */
.woocommerce-form-login input[type="checkbox"],
.oxy-woo-checkout input[type="checkbox"] {
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  opacity: 0.01;
  display: none !important;
}

"display: none !important;" actually hide our checkbox and does not create the custom checkboxes.
Can you please look into asap.

**Comment by KittenCodes** [comment]
@webzstore-sahel Are you able to reproduce this on a [Sandbox site](https://oxygenbuilder.com/try/) so that we can take a look please?

**Comment by webzstore-sahel** [comment]
Hi @KittenCodes,
Sandbox looks good. Because it is working on the default woocommerce template.  In our case, we have customized the checkout layout. We have changed the order of span and input.

`<label class="woocommerce-form__label woocommerce-form__label-for-checkbox checkbox">
									    <span>Ship to a different address?</span>
										<input id="ship-to-different-address-checkbox" class="woocommerce-form__input woocommerce-form__input-checkbox input-checkbox" type="checkbox" name="ship_to_different_address" value="1">
									</label>`

In the above case, it is not working. How can I disable the custom checkout provided by this new version? We can not change the order.
